src/expressions/operators/compares/valueCompare.ts

```diff
@@ -220,7 +302,7 @@ function generateCompareFunction(
 					);
 				};
 			case 'neOp':
-				return (a, b) => {
+				return (a, b, dynamicContext) => {
```

Comment:
Surprised to see this pass the linter. We usually prefixed unused variables with a `_`.

Reply:
We are using `dynamic context` underneath here on line 310